modules/programs/rofi-pass.nix

```
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let cfg = config.programs.rofi.pass;
+in {
+  meta.maintainers = [ maintainers.seylerius ];
+
+  options.programs.rofi.pass = {
+    enable = mkEnableOption "rofi integration with password-store";
+
+    stores = mkOption {
+      type = types.str;
+      default = "";
+      description = "Directory root of your password-store. "
+        + "Separate multiple roots with <literal>:</literal>, "
+        + "as <literal>/foo/bar/baz:/path/to/root2</literal>.";
```

Since multiple roots can be given it makes sense to use a list here. Also, it would be good to use regular multi-line strings in the description to fit the style in the other modules.

```suggestion
      type = types.listOf types.str;
      default = [ ];
      description = ''
        Directory roots of your password-stores.
      '';
```